wps_shared/wps_shared/db/crud/weather_models.py

```diff
@@ -181,7 +181,27 @@ def get_latest_station_model_prediction_per_day(session: Session, station_codes:
 
 
 def get_latest_station_prediction(session: Session, station_codes: List[int], day_start: datetime.datetime, day_end: datetime.datetime):
-    logger.info("Getting data from materialized view.")
+    logger.info("Getting data from weather_station_model_predictions.")
```

Comment:
We might also want to add a doc comment describing the lookup, including the details that we missed before.